### PR TITLE
Fixes lag on new player login

### LINKED ISSUE #14753: mob/new_player/Login() is super laggy
On the profile:
/mob/Login               34.391       39.085       39.086          257 calls

The reason is because it calls ..() (added in https://github.com/tgstation/-tg-station/pull/12869/files#diff-c3d8d983e2c6ebdd2156952858c6907bR2)

Something to do with mob/Login(), the game tries to create a mob at the first location it can Move() to. And the new player fails Move() for every turf or something (I got approximately 600k new_player/Move() calls for each Login() on my profile).

### PR DESCRIPTION
**Pull request by Menshin**
Base `Login()` failed to put location-less mob on the map.

Fixes #14753.

### CODE REVIEW

**Comment by phil235** [comment]
Yep, this fixes the lag.

**Comment by MrStonedOne** [comment]
This isn't quite the *right* fix, but it's a fix.

Really, Login() is not when we should be handling setting loc, New() is.

**Comment by Menshin** [comment]
> Really, Login() is not when we should be handling setting loc, New() is.

Truth

File: code/modules/mob/new_player/new_player.dm [hunk 1]
```diff
@@ -18,6 +18,11 @@
 	tag = "mob_[next_mob_id++]"
 	mob_list += src
 
+	if(length(newplayer_start))
+		loc = pick(newplayer_start)
+	else
+		loc = locate(1,1,1)
```
Comment (YotaXP) [inline code/modules/mob/new_player/new_player.dm]: This stuff should be `..(putTheLocHere)`

Comment (Menshin) [inline code/modules/mob/new_player/new_player.dm]: Wouldn't that add the new_player mob to ```living_mob_list``` ?

**Comment by optimumtact** [comment]
Looks like a transient NPM build failure, fine to just retrigger these failed builds

File: code/modules/mob/new_player/new_player.dm (hunk 1 above)
Comment (YotaXP) [inline code/modules/mob/new_player/new_player.dm]: I... guess it does.  Why isn't that a `/mob/living` thing?

**Comment by Jordie0608** [comment]
Restarted

**Comment by freerealestate** [comment]
i oppose this pr - lag is part of the game - stop removing features

signed, ajfsaslfj

**Pull request merged**